src/python/turicreate/test/test_sarray.py

```diff
@@ -3215,3 +3215,10 @@ def test_filter_by(self):
         self.assertEqual(sorted(x.filter_by(['cat', 'hamster', 'dog', 'fish', 'bird', 'snake'])), ['cat', 'dog'])
         self.assertEqual(sorted(x.filter_by(['cat', 'hamster', 'dog', 'fish', 'bird', 'snake'], exclude=True)), ['cow', 'horse'])
         self.assertEqual(sorted(x.filter_by('dog')), ['dog'])
+
+    def test_abs(self):
```

Comment:
well, if you search `__abs__`, you can move those tests to your `test_abs` section.